soot-infoflow-android/test/soot/jimple/infoflow/android/test/droidBench/FieldSourceTest.java

```diff
@@ -0,0 +1,17 @@
+package soot.jimple.infoflow.android.test.droidBench;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.xmlpull.v1.XmlPullParserException;
+import soot.jimple.infoflow.results.InfoflowResults;
+
+import java.io.IOException;
+
+public class FieldSourceTest extends JUnitTests {
+
+    @Test
+    public void runTestFlowSensitivity1() throws IOException, XmlPullParserException {
+        InfoflowResults res = analyzeAPKFile("FieldSource/angel.numbers.apk");
+        System.out.println(res);
```

Comment:
Please use proper test case checks, i.e., JUnit assertions. If you only print to the command line, the test case will not fail, even if the flow is not found.

Reply:
I solved this one as you suggested.